internal/objects/clusterrole/cluster_role.go

```diff
@@ -167,7 +168,8 @@ func createClusterRole(ctx context.Context, cli client.Client, cr *rbacv1.Cluste
 // updateClusterRoleIfNeeded updates a ClusterRole resource if current does not match desired,
 // using contour to verify the existence of owner labels.
 func updateClusterRoleIfNeeded(ctx context.Context, cli client.Client, contour *operatorv1alpha1.Contour, current, desired *rbacv1.ClusterRole) (*rbacv1.ClusterRole, error) {
-	if objcontour.OwnerLabelsExist(current, contour) {
+	lbls := objcontour.OwnerLabels(contour)
+	if labels.Exist(current, lbls) {
```

Comment:
You've used this pattern quite a bit, but why the temp var? Why not use OwnerLabels directly?
```suggestion
	if labels.Exist(current, objcontour.OwnerLabels(contour)) {
```